proto/bragi/bragi_service.proto

```diff
@@ -17,8 +17,9 @@ import "bragi/valorant.proto";
 
 service Bragi {
 
-  // Matches gRPC unary call returns all planned or currently played matches
-  rpc MatchTimeline(MatchTimelineRequest) returns (MatchTimelineResponse) {}
+  // Matches gRPC stream call returns all planned or currently played matches,
+  // and match updates in realtime when a match changes
```

Comment:
Suggestion: describe also alg. how a client is supposed to maintain updates.

Reply:
Done